packages/core/components/ListPicker/index.tsx

```diff
@@ -132,43 +157,58 @@ export default function ListPicker(props: ListPickerProps) {
                             styles.actionButton
                         )}
                         disabled={!hasSelectedItem}
+                        iconProps={{ iconName: "Delete" }}
                         title={hasSelectedItem ? undefined : "No options selected"}
                         onClick={onDeselectAll}
                     >
                         Reset
                     </ActionButton>
                 </div>
             </div>
-            <List
-                getKey={(item) => String(item.value)}
-                ignoreScrollingState={true}
-                items={filteredItems}
-                onShouldVirtualize={() => filteredItems.length > 100}
-                onRenderCell={(item) =>
-                    item && (
-                        <label className={styles.item}>
-                            <input
-                                className={styles.checkbox}
-                                disabled={props.disabled}
-                                type="checkbox"
-                                role="checkbox"
-                                name={String(item.value)}
-                                value={String(item.value)}
-                                checked={item.selected}
-                                aria-checked={item.selected}
-                                onChange={(event) => {
-                                    if (event.target.checked) {
-                                        onSelect(item);
-                                    } else {
-                                        onDeselect(item);
-                                    }
+            <div className={styles.mainContent}>
```

Comment:
This seems to have helped reduce the times I can reproduce this by a large margin, but I can still make it happen in some rare cases. Might need to move into a different way of rendering this list - I don't think fluentui is good as guessing bounds of submenus [597528c](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/79/commits/597528c4e610e6055be617d1ffd1c8544679d1b9)

Reply:
One possible temporary fix might be to change `height: 75vh;` to `max-height: 75vh;` in ListPicker.module.css since that way it's not fixed to be the full size even if there are only a couple options?

Reply:
Good idea! I'll try this out